### DISCUSSION TITLE
JWS sign issue

### DISCUSSION THREAD

**Discussion by babagay** [kjur/jsrsasign#589] [Q&A]
Hello!
I am trying to sign some payload using a jsrsasign:
var signedJWT = KJUR.jws.JWS.sign("RS256", strJwtHeader, strJwtPayload, privateKey, pass)
but getting error
init failed:this only supports TripleDES
I am newbie in encryption, so totally in stuck with this.
I havent found relevant documentation how to use TripleDES algorythm.
Could someone get me an example? Or hust help to undestand the meaning of that error?

**Comment by kjur** [comment]
Your private key or password seems wrong or not supported by jsrsasign. Can you try with unencrypted PEM private key?

**Comment by babagay** [comment]
Yep, its working with unecrypted key! But are there any tools to decrypt it with your library?

**Comment by kjur** [comment]
It doesn't seems encryption and/or key derivation algorithms in your private are supported in jsrsasign. So there is no way to decrypt your key by jsrsasign. You can use openssl or other tools. Otherwise W3C WebCryptography API may be used.
https://www.w3.org/TR/WebCryptoAPI/
